_includes/video.html

```diff
@@ -1,5 +1,7 @@
 {%- if include.source == "loom" -%}
 {%- assign embed_base_url = "https://www.loom.com/embed" -%}
+{%- else if include.source == "wistia" -%}
```

Comment:
this should be a elsif
```suggestion
{%- elsif include.source == "wistia" -%}
```